tests/test_codex_deferred_settlement.py

```
@@ -0,0 +1,56 @@
+                cli_path,
+                root,
+            )
+            test_structured_background_work_bounds_and_generation_owned_clear(
+                cli_path,
+                root,
+            )
+            test_codex_deferred_settlement_has_single_live_replay_claim(
+                cli_path,
+                root,
+            )
+        except Exception as exc:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)